MIMWebClient/Core/Player/Inventory.cs

```diff
@@ -12,26 +12,19 @@ namespace MIMWebClient.Core.Player
     public class Inventory
     {
 
-        public static void ReturnInventory(List<Item> inventory, Player player)
+        public static void ReturnInventory(ItemContainer inventory, Player player)
         {
-            if (inventory != null)
-            {
-               
-                var inventoryItems = new StringBuilder();;
-                inventoryItems.Append("You are carrying:").AppendLine();
-
-                foreach (var item in player.Inventory.Where(x => x.location == Item.ItemLocation.Inventory))
-                {
-
-                    var result = AvsAnLib.AvsAn.Query(item.name);
-                    string article = result.Article;
-
+            if (inventory != null && inventory.Count > 0)
+            {
+
+                var itemList = new StringBuilder();
+                itemList.Append("You are carrying:").AppendLine();
 
-                    inventoryItems.Append(article.ToUpper() + " " + item.name).AppendLine();
-                }
-                
+                foreach(var item in ItemContainer.List(inventory.Where(x => x.location == Item.ItemLocation.Inventory))){

```

Comment:
Pass true in the itemcontainer list so we get the article showing before the item.